Resources/Locale/en-US/accent/italian.ftl

```diff
@@ -70,28 +74,33 @@ accent-italian-words-21 = nuke
 accent-italian-words-replace-21 = spiciest-a meatball
 
 accent-italian-words-22 = op
+accent-italian-words-22-2 = operative
 accent-italian-words-replace-22 = greek
 
-accent-italian-words-23 = operative
-accent-italian-words-replace-23 = greek
+accent-italian-words-23 = operatives
+accent-italian-words-23-2 = ops
+accent-italian-words-replace-23 = greeks
 
-accent-italian-words-24 = operatives
-accent-italian-words-replace-24 = greeks
+accent-italian-words-24 = sec
+accent-italian-words-24-2 = security
+accent-italian-words-24-3 = secoff
+accent-italian-words-replace-24 = polizia
 
-accent-italian-words-25 = sec
-accent-italian-words-replace-25 = polizia
+accent-italian-words-25 = shitcurity
+accent-italian-words-25-2 = shitsec
+accent-italian-words-replace-25 = carabinieri
```

Comment:
This is a bit bad taste.

Reply:
I didnt add any new ones I just reordered and condensed some of them but I can remove it sure. (it is cutrrently like this ingame).

Reply:
> This is a bit bad taste.

Want me to remove all the bad taste ones that exsisted in the original pr?

I really want this pr to just either be blocked or get merged soon its been a while.